test/services/editions/EditionsTemplatingTest.scala

```diff
@@ -249,6 +253,79 @@ class EditionsTemplatingTest extends FreeSpec with Matchers with OptionValues wi
     }
   }
 
+  object TestCapi extends Capi {
```

Comment:
Moved Editions Templating test file into the correct package (to reduce red flag in IDE).
Merged the 2 Editions Templating test files which were then sitting next to each other.